schema/artwork/context.js

```diff
@@ -46,29 +46,33 @@ const choose = flow(compact, first);
 export default {
   type: ArtworkContextType,
   description: 'Returns the associated Fair/Sale/PartnerShow',
-  resolve: ({ id }) =>
-    Promise
-      .all([
-        gravity('related/sales', { artwork: [id], size: 1, active: false })
```

Comment:
`active: false` doesn't really do anything/I'm not sure the intent? The original endpoint, related/sales, doesn't do anything with this param.